### PR TITLE
ANC Subsequent Visit - Immunizations activity

### PR DESCRIPTION
**Pull request by anvmn**
#261
#269 

### Requires running sanity of Well Child immunisation

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_835

**Comment by anvmn** [comment]
An example where we offer a dose even though 1 year period has not passed, because no doses were given during current pregnancy:

Selection_869

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Measurement/Model.elm [hunk 1]
```diff
@@ -407,3 +409,63 @@ type InvokationModule
...
+
+
+type alias VaccinationForm msg =
+    { administeredDoses : Maybe (EverySet VaccineDose)
+    , administeredDosesDirty : Bool
+    , administrationDates : Maybe (EverySet NominalDate)
+
+    -- This is the note for suggesed dose for encounter.
+    -- There are situations where there will be no suggested dose,
+    -- due to ability to uodate previous doses.
```
Comment (nedSaf) [inline client/src/elm/Measurement/Model.elm]:
```suggestion
    -- due to the ability to update previous doses.
```

**Review by nedSaf — APPROVED** [review]
Looks good, one typo.

**Pull request merged**